localstack/services/resourcegroupstaggingapi/rgta_listener.py

```diff
@@ -1,28 +0,0 @@
-from requests.models import Request
-
-from localstack.services.generic_proxy import ProxyListener
-from localstack.utils.aws.aws_responses import MessageConversion
-
-
-class ProxyListenerRGSA(ProxyListener):
-    def forward_request(self, method, path, data, headers):
-        if method == "POST" and path == "/":
-            data = MessageConversion._reset_account_id(data)
-            return Request(data=data, headers=headers, method=method)
-
-        return True
-
-    def return_response(self, method, path, data, headers, response):
-        if response.content:
-            # fix hardcoded account ID in ARNs returned from this API
-            MessageConversion.fix_account_id(response)
```

Comment:
Probably already been fixed upstream, but can you confirm that these patches are no longer required? (looks like not all of them are necessarily covered by our integration tests)